src/fri/naive.md

```diff
@@ -0,0 +1,157 @@
+
+
+*For simplicity, the random number $\beta$ will be assumed as 1 in this post. We would touch the implication behind this random number in another post that explains the commitment.*
+
+$p_0$ is the original polynomial, and $p_2$ is a constant polynomial that can't be folded anymore. 
+
+These folded polynomials can be traced back to their source polynomials. Thus, the evaluations in a folded polynomial can be accumulated from the corresponding source polynomial. 
+
+These relationships can serve as a proof for verifier to the consistency among the evaluations. The consistency check implies if there is a $p_0$ exists that the prover claimed.
+
+With the same folding setup, the final polynomial will be always the same value. Verifier can check against the same final value against the reduced values from different evaluation points.
+
+If the final polynomial can be derived after expected rounds of folding, it implies the $p_0$ has an expected upper bound degree. In other words, if the final $p_k$ can be derived after $k=log(d)$ rounds of folding from $p_0$ which is of degree $d$, then $p_0$ has upper bound degree $d$.
+
+More formally, there are two properties:
+- the number of rounds $k=log(d)$ to derive the final polynomial implies the expected upper bound degree $d$
+- the same final value can be re-used to check consistency of the evaluation claims across layers
+
+We will refer this folding example again with some actual evaluations to demonstrate the interesting relationship among them.
+
+## What is the problem?
+
+Using the notation from [what is proving](https://nmohnblatt.github.io/zk-jargon-decoder/intro_to_zk/what_is_proving.html), we define the following:
+
+$$
+\mathcal{L}(\mathcal{R}_{\text{consistent-layers}})
+$$
+
+for instance, $\left(a_1, w_1\right) \in \mathcal{R}$, where $a_1$ can represent a FRI query and constant value from $p_2$, while $w_1$ represents the corresponding evaluations across layers.
+
+So the problem is to do the check to see if $a_1$ is in $\mathcal{L}$. In other words, the verifier checks if the prover can provide $w_1$ satisfying the query $a_1$.
+
+## naive FRI
+
+Let's assume prover is honest guy in this naive version of FRI. The goal is for verifier to do fast low degree test using the proof from prover. 
+
+### Hard honest work by prover
+
+Prover can use the folded polynomial to obtain the evaluations for each layers. These evaluations of layers can serve as a proof for the verifier to sample for their consistency across layers.
+
+Using the same polynomial example shown earlier, here are two simple calculation examples.
+
+For evaluation point at *1*:
+
+$$
+\begin{align} 
+\text{Layer 1:} & \\
+x_0&=1 \\
+p_0(x_0) &= 1 + 2x_0 + 3x_0^2 \\ 
+p_0(1) &= 6 \\ 
+p_0(-1) &= 2 \\\\
+\text{Layer 2:} & \\
+x_1&=x_0^2 \\
+p_1(x_1) &= 3 + 3x_1 \\
+p_1(1^2) &= 6 \\ 
+p_1(-1^2) &= 0 \\\\
+\text{Layer 3:} & \\
+x_2&=x_1^2 \\
+p_2(x_2) &= 6 \\
+p_2((1^2)^2) &= 6 \\\\
+\end{align}
+$$
```

Comment:
a think a list would render better here as well, doesn't render super well at least on github

Reply:
also I don't understand why you evaluate at 1 and -1 here? Aren't you just doing the reduction at this point? Why evaluate?

Reply:
My understanding is the prover need to evaluate the full domain and then commit the evaluations. Evaluating at 1 and -1 is an example of evaluating at point 1 and its symmetric point -1. Here I try to explain how these evaluations are done at the prover side.

Reply:
ah, I guess it is part of committing to the polynomial at each layer (but you were not talking about commitments so far). Grouping v and -v under the same leaf is an optimization :o